packaging/build_deb

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#
+# Builds the libsxg deb packages. The script must be run at the root
+# directory of the libsxg source tree. The deb packages will be copied to
+# the current directory.
+#
+# This script is written mainly for CI.
+
+set -ex
+
+readonly SRCDIR="${PWD}"
+readonly TARGET_DIR="${1}"
+readonly WORKDIR="$(mktemp -d --tmpdir=/tmp build_deb.XXXXXX)"
```

Comment:
Just `--tmpdir` is enough; the argument (`/tmp`) is not needed. It's possibly better as it will use the default temporary directory whatever it is.